examples/basic/basicSend.js

```diff
@@ -10,15 +10,15 @@ message.subject = "Sending A Test Message (Basic Send)";
 message.htmlBody = "<html><body><h1>Sending A Test Message</h1><p>This is the Html Body of my message.</p></body></html>";
 message.textBody = "This is the Plain Text Body of my message.";
 
-message.from = new EmailAddress("from@example.com");
-message.replyTo = new EmailAddress("replyto@example.com");
+message.from = new EmailAddress("mirriam.ronoh@socketlabs.com");
```

Comment:
Please revert to previous example and do not include personal email addresses in any examples.